client/lib/domains/registration/availability-messages.js

```diff
@@ -133,6 +133,16 @@ function getAvailabilityNotice( domain, error ) {
 			);
 			break;
 
+		case domainAvailability.TRANSFER_PENDING:
+			message = translate(
+				'{{strong}}%(domain)s{{/strong}} is already connected to a WordPress.com site.',
```

Comment:
Not sure if this is the best explanation of this state - if I were a confused user that wanted to "restart" a transfer or try mapping the domain instead, I would be even more confused by this error.

Reply:
Agree. Still thinking with @danhauk what to use there.

Reply:
@klimeryk @aidvu I just pushed up a change to that message to make it more clear as well as provide a support link. The rest LGTM.